openmetadata-ui/src/main/resources/ui/src/constants/Mydata.constants.ts

```diff
@@ -13,9 +13,7 @@
 
 import { SearchIndex } from '../enums/search.enum';
 
-export const myDataSearchIndex =
-  // eslint-disable-next-line max-len
-  `${SearchIndex.TABLE},${SearchIndex.TOPIC},${SearchIndex.DASHBOARD},${SearchIndex.PIPELINE},${SearchIndex.MLMODEL},${SearchIndex.STORED_PROCEDURE},${SearchIndex.DASHBOARD_DATA_MODEL},${SearchIndex.DATABASE},${SearchIndex.DATABASE_SCHEMA},${SearchIndex.DATABASE},${SearchIndex.DATABASE_SERVICE},${SearchIndex.MESSAGING_SERVICE},${SearchIndex.PIPELINE_SERVICE},${SearchIndex.SEARCH_SERVICE},${SearchIndex.DASHBOARD_SERVICE},${SearchIndex.ML_MODEL_SERVICE},${SearchIndex.STORAGE_SERVICE},${SearchIndex.SEARCH_INDEX}` as SearchIndex;
+export const MY_DATA_SEARCH_Index = SearchIndex.ALL;
```

Comment:
We don't need this const, We can directly use SearchIndex.ALL. It will be more readable